smartsim/settings/base.py

```diff
@@ -272,6 +486,19 @@ def format_run_args(self):
             formatted.append(str(value))
         return formatted
 
+    def format_env_vars(self):
```

Comment:
I think we should just return a string that would work with the ``env`` command here. this is overridden in every class anyway.

Reply:
Implemented in [e887de9](https://github.com/CrayLabs/SmartSim/pull/170/commits/e887de9b7bfd74f21575428bb3ff2c213ef5f6f8)

Just want to confirm that is okay that function signature are changed by inherited classes (for instance `MpirunSettings.format_env_vars` returns a `list[str]`).

Reply:
signatures unified in [2e20b85](https://github.com/CrayLabs/SmartSim/pull/170/commits/2e20b858b143dc048a2806a25b676c0892159c9b)

going to give this a final pass on monday before merging